### DISCUSSION TITLE
Add Text Completion in TextEdit Like in some text/code editors

### DISCUSSION THREAD

**Discussion by Karl-Za-Great** [godotengine/godot-proposals#5738] [2D]
Maybe ya'll can add text completion feature in TextEdit node where when you type a word a list of related words show up near the cursor

**Comment by ghost** [comment]
> Maybe ya'll can add text completion feature in TextEdit node where when you type a word a list of related words show up near the cursor

Please add the text align feature..

**Comment by awardell** [comment]
The text editor has text completion. Can you give a more elaborate example of what you feel it is missing? Maybe telling us what other "text/code editors" you are referring to would help.

**Reply by Karl-Za-Great** [reply]
Like godot script editor, when you type fun there is a list of words like "func" etc. 

And also how to use that text completion thingy?

**Reply by Calinou** [reply]
> Like godot script editor, when you type fun there is a list of words like "func" etc.

Using a script, you can add completion options to any TextEdit node. See this for an example: https://github.com/gdscript-online/gdscript-online.github.io/pull/7
